src/Schema/Definitions/UniqueWhereDefinition.php

```diff
@@ -0,0 +1,124 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Umbrellio\Postgres\Schema\Definitions;
+
+use Illuminate\Support\Fluent;
+
+class UniqueWhereDefinition extends Fluent
```

Comment:
Im not sure, that this class should be in this namespace

Reply:
fixed